File: clients/dotnet/enums/BatchStatus.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace Avalara.AvaTax.RestClient
{
    /// <summary>
    /// 
    /// </summary>
    public enum BatchStatus
    {
        /// <summary>
        /// 
        /// </summary>
        Waiting,

        /// <summary>
        /// 
        /// </summary>
        SystemErrors,

        /// <summary>
        /// 
        /// </summary>
        Cancelled,

        /// <summary>
        /// 
        /// </summary>
        Completed,

        /// <summary>
        /// 
        /// </summary>
        Creating,

        /// <summary>
        /// 
        /// </summary>
        Deleted,

        /// <summary>
        /// 
        /// </summary>
        Errors,

        /// <summary>
        /// 
        /// </summary>
        Paused,

        /// <summary>
        /// 
        /// </summary>
        Processing,


    }
}

File: clients/php/enums/BatchStatus.php
<?php 
namespace Avalara;
/*
 * AvaTax API Client Library
 *
 * (c) 2004-2016 Avalara, Inc.
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 *
 * @category   AvaTax client libraries
 * @package    Avalara.AvaTaxClient
 * @author     Ted Spence <ted.spence@avalara.com>
 * @author     Bob Maidens <bob.maidens@avalara.com>
 * @copyright  2004-2016 Avalara, Inc.
 * @license    https://www.apache.org/licenses/LICENSE-2.0
 * @version    2.16.12-30
 * @link       https://github.com/avadev/AvaTaxClientLibrary
 */


/**
 * 
 */
class BatchStatus
{

    const Waiting = "Waiting";
    const SystemErrors = "SystemErrors";
    const Cancelled = "Cancelled";
    const Completed = "Completed";
    const Creating = "Creating";
    const Deleted = "Deleted";
    const Errors = "Errors";
    const Paused = "Paused";
    const Processing = "Processing";
}
